bedrock/careers/templates/careers/internships.html

```diff
@@ -12,14 +12,14 @@
 {% block body_class %}internships-page{% endblock %}
 
 {% block careers_content %}
-<section class="mzp-l-content mzp-c-hero mzp-t-dark">
```

Comment:
Note for reviewer. This page is temporarily being redirected back to the /careers home page. I'm not sure if it's still needed, but appears to be in a somewhat half finished state perhaps (I'm not too sure).

I had to remove the redirect locally and define a URL in order to view the changes here.

Reply:
@reemhamz probably has the best insight here